pkg/registry/core/pod/rest/subresources.go

```diff
@@ -242,7 +242,13 @@ func (r *PortForwardREST) Connect(ctx context.Context, name string, opts runtime
 	if err != nil {
 		return nil, err
 	}
-	return newThrottledUpgradeAwareProxyHandler(location, transport, false, true, responder), nil
+	handler := newThrottledUpgradeAwareProxyHandler(location, transport, false, true, responder)
+	if utilfeature.DefaultFeatureGate.Enabled(features.PortForwardWebsockets) {
+		maxBytesPerSec := capabilities.Get().PerConnectionBandwidthLimitBytesPerSec
+		tunnelingHandler := translator.NewTunnelingHandler(handler, maxBytesPerSec)
+		handler = translator.NewTranslatingHandler(handler, tunnelingHandler, wsstream.IsWebSocketRequestWithPortForwardProtocol)
```

Comment:
nit: if tunnelingHandler is delegating all data flow serially to handler, I *think* we can just rely on its maxBytesPerSec enforcement